src/core/client/ui/components/Timestamp/Timestamp.css

```diff
@@ -0,0 +1,4 @@
+.root {
+  composes: button from "talk-ui/shared/typography.css";
```

Comment:
We need to add typography styling from Sam or if already available from the Stylesheet

Reply:
👍